### PR TITLE
[BUG] Hide login button when setting is disabled

### PR DESCRIPTION
**Pull request by rafaelks**
@RocketChat/ios

### CODE REVIEW

File: Rocket.Chat/Controllers/Auth/AuthTableViewController.swift [hunk 1]
```diff
@@ -124,10 +124,13 @@ class AuthTableViewController: BaseTableViewController {
...
         super.viewDidLoad()
+
         title = serverURL.host
 
         guard let settings = serverPublicSettings else { return }
 
+        emailAuthRow.loginButton.isHidden = !settings.isUsernameEmailAuthenticationEnabled
+
         if !settings.isUsernameEmailAuthenticationEnabled {
             emailAuthRow.registerButton.isHidden = true
```
Comment (filipealva) [inline Rocket.Chat/Controllers/Auth/AuthTableViewController.swift]: @rafaelks we could put the new line inside the `if` block. That way we would evaluate `settings.isUsernameAuthenticationEnabled` just once.

**Comment by filipealva** [comment]
@rafaelks actually if `isUsernameAuthenticationEnabled` we shouldn't even show the `emailAuthRow`  and neither the separator of the login services section

**Comment by rafaelks** [comment]
@filipealva Done!

**Comment by filipealva** [comment]
@rafaelks Nice! but just hiding the cell still let us considering its height on uitableview's content size, which means that if you have a server with a certain number of login services you'll have an issue with an extra offset

**Comment by rafaelks** [comment]
@filipealva I see... this is not using AutoLayout, so when the registerButton was hidden it was already having a white space. Will make some adjustments here.

**Pull request merged**